docs/features/networking.md

```diff
@@ -0,0 +1,56 @@
+# Networking and communicating with containers
+
+## Exposing container ports to the host
+
+It is common to want to connect to a container from your test process, running on the test 'host' machine.
+For example, you may be testing some code that needs to connect to a backend or data store container.
+
+Generally, each required port needs to be explicitly exposed. For example, we can specify one or more ports as follows:
+
+<!--codeinclude-->
+[Exposing ports](../../docker_test.go) inside_block:exposePorts
+<!--/codeinclude-->
+
+Note that this exposed port number is from the *perspective of the container*. 
+
+*From the host's perspective* Testcontainers actually exposes this on a random free port.
+This is by design, to avoid port collisions that may arise with locally running software or in between parallel test runs.
+
+Because there is this layer of indirection, it is necessary to ask Testcontainers for the actual mapped port at runtime.
+This can be done using the `getMappedPort` method, which takes the original (container) port as an argument:
```

Comment:
```suggestion
This can be done using the `MappedPort` function, which takes the original (container) port as an argument:
```

Not sure what the correct Go lingo is here, I think they are functions, not methods?

Reply:
Yeah, functions. I missed this while copy&paste'ing from Java. BTW, the lingo there was so generic and not coupled to the java that it directly applied to Go. Great!!